beacon-chain/p2p/service.go

```diff
@@ -43,6 +43,10 @@ var _ runtime.Service = (*Service)(nil)
 // defined below.
 var pollingPeriod = 6 * time.Second
 
+// When looking for new nodes, if not enough nodes are found,
+// we stop after this amount of iterations.
+var batchSize = 40_000
```

Comment:
40000 seems like a pretty big amount, was 2000 too small previously ?

Reply:
Actually I moved from 2.000 to 40.000 to avoid too frequent logging.

Fixed in https://github.com/prysmaticlabs/prysm/pull/14446/commits/7f70bece39b0926559f0fff733aae7d866120a92.